File: JavaScript_Courses/JavaScript_Basics/Preparation/03_Conditional_Statements_Advanced_Exercise/02_Summer_Outfit.js
function summerOutfit(input) {
    let degrees = Number(input[0]);
    let timeOfDay = input[1];

    let clothes;
    let shoes;

    if (timeOfDay == "Morning") {
        if (degrees >= 10 && degrees <= 18) {
            clothes = "Sweatshirt";
            shoes = "Sneakers";
        } else if (degrees > 18 && degrees <= 24) {
            clothes = "Shirt";
            shoes = "Moccasins";
        } else if (degrees >= 25) {
            clothes = "T-Shirt";
            shoes = "Sandals";
        }
    } else if (timeOfDay == "Afternoon") {
        if (degrees >= 10 && degrees <= 18) {
            clothes = "Shirt";
            shoes = "Moccasins";
        } else if (degrees > 18 && degrees <= 24) {
            clothes = "T-Shirt";
            shoes = "Sandals";
        } else if (degrees >= 25) {
            clothes = "Swim Suit";
            shoes = "Barefoot";
        }
    } else if (timeOfDay == "Evening") {
        if (degrees >= 10 && degrees <= 18) {
            clothes = "Shirt";
            shoes = "Moccasins";
        } else if (degrees > 18 && degrees <= 24) {
            clothes = "Shirt";
            shoes = "Moccasins";
        } else if (degrees >= 25) {
            clothes = "Shirt";
            shoes = "Moccasins";
        }
    }

    console.log(`It's ${degrees} degrees, get your ${clothes} and ${shoes}.`);
}

// summerOutfit(["16", "Morning"]);

File: Python_Courses/Python_Basics/Python_PB_2024_10/03_Conditional_Statements_Advanced_Exercise/02_Summer_Outfit.py
degrees = int(input())
daytime = input()  # "Morning", "Afternoon", "Evening"

outfit = ''
shoes = ''
# 10 <= градуси <= 18
# 18 < градуси <= 24
# градуси >= 25

if daytime == 'Morning':
    if 10 <= degrees <= 18:
        outfit = 'Sweatshirt'
        shoes = 'Sneakers'
    elif 18 < degrees <= 24:
        outfit = 'Shirt'
        shoes = 'Moccasins'
    elif degrees >= 25:
        outfit = 'T-Shirt'
        shoes = 'Sandals'

elif daytime == 'Afternoon':
    if 10 <= degrees <= 18:
        outfit = 'Shirt'
        shoes = 'Moccasins'
    elif 18 < degrees <= 24:
        outfit = 'T-Shirt'
        shoes = 'Sandals'
    elif degrees >= 25:
        outfit = 'Swim Suit'
        shoes = 'Barefoot'

elif daytime == 'Evening':
    # outfit = 'Shirt'
    # shoes = 'Moccasins'
    if 10 <= degrees <= 18:
        outfit = 'Shirt'
        shoes = 'Moccasins'
    elif 18 < degrees <= 24:
        outfit = 'Shirt'
        shoes = 'Moccasins'
    elif degrees >= 25:
        outfit = 'Shirt'
        shoes = 'Moccasins'

print(f'It\'s {degrees} degrees, get your {outfit} and {shoes}.')
